core/indigo-core/layout/src/pathway_layout.cpp

```diff
@@ -142,7 +144,62 @@ void PathwayLayout::buildLayoutTree()
                 }
             }
         }
+
+        auto totalLines = reactionNode.name_text.size() + reactionNode.conditions_text.size();
+        if (totalLines && currentLayoutItem.children.size() > 1)
+        {
+            if (totalLines > MIN_LINES_COUNT)
+                totalLines = MIN_LINES_COUNT;
+
+            auto totalHeight = std::accumulate(currentLayoutItem.children.begin(), currentLayoutItem.children.end(), 0.0f,
+                                               [](float summ, const PathwayLayoutItem* item) { return summ + item->height; });
+            totalHeight -= currentLayoutItem.children.front()->height / 2;
+            totalHeight -= currentLayoutItem.children.back()->height / 2;
+            totalHeight /= 2.0f;
```

Comment:
add constant for 2

Reply:
2 is not a magic number. There is no need to create a constant for 2.